uber/models.py

```diff
@@ -1669,6 +1669,7 @@ class ApprovedEmail(MagModel):
 
 class Email(MagModel):
     fk_id   = Column(UUID, nullable=True)
+    email_id = Column(UnicodeText)
```

Comment:
This line is a good example of why ``ident`` might be a better term.  The ``Email`` database table will now have ``email.id`` and ``email.email_id`` columns which are separate things.  Whereas ``email.id`` and ``email.ident`` is probably/hopefully a little less confusing.